### PR TITLE
Added to Favorites popup. Changed favorites star to solid gold

### PR DESCRIPTION
**Pull request by myo3**

### CODE REVIEW

**Comment by say25** [comment]
Missing at commit to link all the files together

**Comment by say25** [comment]
Maybe we should have a removed from favorites as well? thoughts @acheng96

File: Eatery/Eat Now/Detail View Controller/MenuViewController.swift [hunk 1]
```diff
@@ -277,6 +287,15 @@ class MenuViewController: UIViewController, MenuButtonsDelegate, TabbedPageViewC
     
     func favoriteButtonPressed() {
         delegate?.favoriteButtonPressed()
+        if eatery.favorite == true{
```
Comment (anniexcheng) [inline Eatery/Eat Now/Detail View Controller/MenuViewController.swift:290]: Just user eatery.favorite to avoid redundancy

**Comment by anniexcheng** [comment]
I agree with Steven. @myo3, could you add in a 'Removed from Favorites' popup when a eatery is removed from favorites?

**Comment by say25** [comment]
This would probably have the unfilled white star and "Removed from favorites" as text

**Comment by myo3** [comment]
@acheng96 @say25 alright I'll get on it!

**Comment by lucasderraugh** [comment]
Sorry, didn't realize this was still under operation. @myo3 could you start any new work off the new master branch? I made changes that move the logic from the VC into the view and I'd like to avoid display logic being thrown into the VC.

**Comment by myo3** [comment]
Yeah it's not a problem

**Pull request merged**